polars/polars-lazy/polars-plan/src/logical_plan/builder.rs

```diff
@@ -403,6 +420,12 @@ impl LogicalPlanBuilder {
                 &self.0,
                 into
             );
+            if rewritten.is_empty() {
+                let msg = "The predicate expanded to zero expressions. \
+                This may for example be caused by a regex not matching column names or\
```

Comment:
Missing space here 😄 

```suggestion
                This may for example be caused by a regex not matching column names or \
```